examples/scalajs-react/generated/todomvc-launcher.js

```diff
@@ -0,0 +1,2 @@
+((typeof global === "object" && global &&
+         global["Object"] === Object) ? global : this)["todomvc"]["Main"]().main();
```

Comment:
Still we want tab indentation in JS files. Also, as I told you earlier please use our javascript style guide.